internal/clientconn/conn.go

```diff
@@ -36,10 +36,9 @@ import (
 
```

Comment:
Couldn't we change our `--handler` flag in `cmd/ferretdb/main.go` to `--backend`?

Reply:
Not in this PR